Content.Server/Maps/GameMapManager.cs

```diff
@@ -35,6 +38,23 @@ public void Initialize()
                 throw new ArgumentException($"Unknown map prototype {value} was selected!");
         }, true);
         _configurationManager.OnValueChanged(CCVars.GameMapForced, value => _currentMapForced = value, true);
+        _configurationManager.OnValueChanged(CCVars.GameMapRotation, value => _mapRotationEnabled = value, true);
+        _configurationManager.OnValueChanged(CCVars.GameMapQueueDepth, value =>
```

Comment:
IMO "MapQueueDepth" seems like a confusing name to me. It makes me thing people are voting for and queueing up future maps, when its more of a map-memory thing.